generated_api_shadow/envoy/extensions/filters/http/local_ratelimit/v3/local_rate_limit.proto

```diff
@@ -0,0 +1,135 @@
+syntax = "proto3";
```

Comment:
The `generated_api_shadow` directory was removed in #18091.
Please remove this file

Reply:
Ok, recent commits have been merged.